Workday/canvas-kit
# fix: Set Pagination GoTo TextInput to use current page as initial value

## Summary

Fixes: #1929

## Release Category
Components

---

## Checklist

- [x] MDX documentation adheres to Canvas Kit's [standard MDX template](https://github.com/Workday/canvas-kit/discussions/1131)

## For the Reviewer


## Where Should the Reviewer Start?

`modules/react/pagination/lib/Pagination/GoTo/useGoToForm.tsx`

## Areas for Feedback? (optional)


## Testing Manually

- Visit the Pagination GoTo Form example
- See the initial value of the input is set to the current page (1)
- See the value or the input change by using the Pagination controls.

## Screenshots or GIFs (if applicable)

![a demo of the Pagination GoTo TextInput with the updated behavior](http://g.recordit.co/ANk6nzpRRm.gif)

## Thank You Gif (optional)

![a cat reading a book on military strategy](https://media.giphy.com/media/NFA61GS9qKZ68/giphy.gif)
